eras/shelley-ma/test-suite/cardano-ledger-shelley-ma-test.cabal

```
@@ -41,16 +41,16 @@ library
     build-depends:
         base >=4.14 && <4.19,
         bytestring,
-        cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.2,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.3,
+        cardano-ledger-binary:{cardano-ledger-binary, testlib} ^>=1.3,
+        cardano-ledger-core:{cardano-ledger-core, testlib} ^>=1.10,
         cardano-ledger-pretty,
-        cardano-ledger-allegra:{cardano-ledger-allegra, testlib} >=1.2 && <1.3,
-        cardano-ledger-mary:{cardano-ledger-mary, testlib} ^>=1.4,
+        cardano-ledger-allegra:{cardano-ledger-allegra, testlib} ^>=1.3,
+        cardano-ledger-mary:{cardano-ledger-mary, testlib} ^>=1.5,
         cardano-slotting,
         containers,
         hashable,
-        cardano-ledger-shelley-test >=1.1,
-        cardano-ledger-shelley:{cardano-ledger-shelley, testlib} >=1.6 && <1.10,
+        cardano-ledger-shelley-test ^>=1.3.0.1,
```

Nothing changed in `cardano-ledger-shelley-test` that is now required here. A good rule to follow is that if you find yourself adding a bound to cabal file on a patch version (eg. `>=x.y.0.1` or `<x.y.z.3`) you are doing something wrong. Because by definition patch version should not have any visible changes.
```suggestion
        cardano-ledger-shelley-test >=1.1,
```